app/controllers/middleware_server_controller.rb

```diff
@@ -293,7 +293,7 @@ def run_operation
   private
 
   def textual_group_list
-    [%i(properties), %i(relationships smart_management)]
+    [%i(properties smart_management), %i(compliance relationships)]
```

Comment:
I think the order is inconsistent with other similar screens.
compliance should be on the left with properties, smart management on the right with relationships.
for example:
https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/container_node_controller.rb#L16
https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/container_image_controller.rb#L11
unless there are some documented guidelines, and the above examples actually got it wrong :)

Reply:
cc @ohadlevy @martinpovolny @serenamarie125  are there guidelines for the order of the sections in summary pages? I think that without validation/guidelines, there's a big chance of getting a lot of different looking screens that should have a unified look n feel

Reply:
Well I tried to make it as even as possible (both sides) and I don't think it's something set in stone, since VMs have compliance history on right. For me it just felt better option 1 (looks slightly even than option 2), but if you prefer option 2, no objections I'll just swap them.
### Option 1 compliance on right 
![screenshot from 2017-11-02 15-14-53](https://user-images.githubusercontent.com/3439771/32330522-a732333c-bfe0-11e7-9081-e07265a0c9b6.png)

### Option 2 Compliance on left
![screenshot from 2017-11-02 15-14-27](https://user-images.githubusercontent.com/3439771/32330516-a416b16e-bfe0-11e7-8363-e9236b843a91.png)

Reply:
I agree that your proposal looks better. it is not about my own preference, but rather if there is a standard cross miq that we should follow. lets see what core team says and do accordingly.
if there is no standard, your suggestion should stay :)

Reply:
For normal in other views the left side is more longer than the right side. looks good for me the option 1 too.